tests/unit_tests/test_wave_07.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from swap_meet.vendor import Vendor
+from swap_meet.item import Item
+from swap_meet.electronics import Electronics
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
```

Comment:
A whole new file of tests! You spoil us! 

Great job using a flexible,`item in person.inventory` syntax here instead of doing a stricter `list_of_items[index] == item` as well!